ProjFS.Mac/PrjFSKextTests/HandleOperationTests.mm

```diff
@@ -191,6 +195,8 @@ - (void) testNonEmptyFileDoesNotHydrate {
         KAUTH_VNODE_WRITE_DATA,
         KAUTH_VNODE_EXECUTE,
         KAUTH_VNODE_DELETE,
+        KAUTH_VNODE_APPEND_DATA,
+        KAUTH_VNODE_ADD_SUBDIRECTORY,
```

Comment:
Similar comment here, it looks a bit strange to have `KAUTH_VNODE_ADD_SUBDIRECTORY` for a file test

Reply:
(this comment applies to all of the file specific tests)

Reply:
I'll fix that up, I had it in only because the code is the same, but you're correct this is a case that shouldn't happen